lib/config-generator.js

```diff
@@ -40,65 +52,15 @@ export class ConfigGenerator {
      * @returns {void}
      */
     async prompt() {
-        const questions = [
-            {
-                type: "select",
-                name: "purpose",
-                message: "How would you like to use ESLint?",
-                initial: 1,
-                choices: [
-                    { message: "To check syntax only", name: "syntax" },
-                    { message: "To check syntax and find problems", name: "problems" }
-                ]
-            },
-            {
-                type: "select",
-                name: "moduleType",
-                message: "What type of modules does your project use?",
-                initial: 0,
-                choices: [
-                    { message: "JavaScript modules (import/export)", name: "esm" },
-                    { message: "CommonJS (require/exports)", name: "commonjs" },
-                    { message: "None of these", name: "script" }
-                ]
-            },
-            {
-                type: "select",
-                name: "framework",
-                message: "Which framework does your project use?",
-                initial: 0,
-                choices: [
-                    { message: "React", name: "react" },
-                    { message: "Vue.js", name: "vue" },
-                    { message: "None of these", name: "none" }
-                ]
-            },
-            {
-                type: "select",
-                name: "language",
-                message: "Does your project use TypeScript?",
-                choices: [
-                    { message: "No", name: "javascript" },
-                    { message: "Yes", name: "typescript" }
-                ],
-                initial: 0
-            },
-            {
-                type: "multiselect",
-                name: "env",
-                message: "Where does your code run?",
-                hint: "(Press <space> to select, <a> to toggle all, <i> to invert selection)",
-                initial: 0,
-                choices: [
-                    { message: "Browser", name: "browser" },
-                    { message: "Node", name: "node" }
-                ]
-            }
-        ];
+        Object.assign(this.answers, await enquirer.prompt(langQuestions));
```

Comment:
didn't use `skip` - as there is a known issue when using multi-selection: https://github.com/enquirer/enquirer/issues/298